source/docs/zero-to-robot/step-3/radio-programming.rst

```diff
@@ -2,198 +2,112 @@
 
-## Install the Software
+## Access Point Radio Configuration
 
-.. image:: images/radio-programming/radio-installer.png
-  :alt: The radio configuration installer .exe file in windows explorer.
+Follow all of the same steps as the robot radio configuration instead choosing :guilabel:`Access Point Mode` at the top of the configuration page.
 
-Double click on ``FRC_Radio_Configuration_VERSION.exe`` to launch the installer. Follow the prompts to complete the installation.
+## Alternative Setup Discussion
 
-Part of the installation prompts will include installing Npcap if it is not already present. The Npcap installer contains a number of checkboxes to configure the install. You should leave the options as the defaults.
+### Optimal Setup: Two VH-109 Radios
 
-## Launch the software
+For the best experience and to closely simulate field conditions, it is strongly recommended that your team uses two VH-109 radios during testing and preparation. This dual-radio setup mirrors the competition environment, ensuring your robot operates under realistic network conditions. Additionally, having two radios allows you to fully leverage the high-speed, low-latency communication provided by the 6GHz band, which is crucial for optimal robot performance in high-stakes scenarios.
 
-.. image:: images/radio-programming/radio-launch.png
-  :alt: The FRC Radio Configuration Utility in the start menu.
+### Only 1 VH-109 radio
 
-Use the Start menu or desktop shortcut to launch the program.
+If your team has access to only one VH-109 radio, there are still viable options to continue testing and preparing for competition. However, these setups require additional considerations:
 
-.. note:: If you need to locate the program, it is installed to ``C:\Program Files (x86)\FRC Radio Configuration Utility``. For 32-bit machines the path is ``C:\Program Files\FRC Radio Configuration Utility``
+#### Use an Old Radio for Testing
 
-## Allow the program to make changes, if prompted
+If your team still has an older radio from a previous season, it can serve as a temporary substitute for a second VH-109. In this case, you should:
 
-.. image:: images/radio-programming/allow-changes.png
-  :alt: User Account Control dialog that pops up when running the config utility.
+Reserve a spot on your robot specifically for the VH-109 radio to ensure seamless integration during competition.
 
-A prompt may appear about allowing the configuration utility to make changes to the computer. Click :guilabel:`Yes` if the prompt appears.
+Be prepared to connect devices via a network switch if the older radio does not provide enough Ethernet ports. This may add complexity but ensures all devices are networked properly during testing.
 
-## Select the network interface
+Advantages:
 
-.. image:: images/radio-programming/select-network-connection.png
-  :alt: Each of the Parts of the Network Interfaces selection pop up.
+Maintains the ability to simulate dual-radio setups with some fidelity.
```

Comment:
Not sure what this means? I was figuring the recommendation of using an OM5P would be put it on the robot as and AP the same as teams had done before?

Reply:
Yes this comment was old.  Removed.